docs/man/nng_device.3.adoc

```diff
@@ -11,7 +11,7 @@
 
 == NAME
 
-nng_sendmsg - send message
+nng_device - send message
```

Comment:
Well this was a total fail on my part.  But let's change the synopsis here too..

nng_device - message forwarding device